lang_qc/endpoints/inbox.py

```diff
@@ -69,12 +76,235 @@ def get_inbox(
         )
     )
 
-    results: List[PacBioRunWellMetrics] = db_session.execute(stmt).scalars().all()
+    return db_session.execute(stmt).scalars().all()
+
+
+def extract_well_label_and_run_name_from_state(state: QcState):
```

Comment:
Yes we could, but then we would have to make the decision to maintain the DB schema in Python code, rather than in SQL. That would come with the bonus of always having schema in sync with the server code.
If we do keep the DB defined from SQL, then we would need to somehow maintain a set of patches on top of `sqlacodegen`-generated ORM classes.
Otherwise, I imagine we should keep this sort of thing separate